lambdaNotes/src/components/Note.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import axios from "axios";
+import EditNote from "./EditNote";
+import "../App.css";
+import {
+  LeftBar,
+  StyledLink,
+  ModalH3,
+  TitleH1,
+  NoteCardContainer,
+  NoteTitle,
+  NoteP,
+  EditNoteLink,
+  DeleteNoteLink,
+  SignOut
+} from "../Styles";
+
+class Note extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      editing: false,
+      title: this.props.notes.title,
+      textBody: this.props.notes.textBody
+    };
+  }
+
+  editNote = event => {
+    event.preventDefault();
+    this.setState({ editing: true });
+  };
+
+  deleteNote = event => {
+    event.preventDefault();
+    const url = `https://fe-notes.herokuapp.com/note/delete/${
+      this.state.note._id
+    }`;
+    axios
+      .delete(url)
+      .then(response => {
+        console.log(response);
+        this.props.updateDeleted(this.state.note._id);
+      })
+      .catch(error => {
+        console.error(error);
+      });
+    this.props.history.push("/");
```

Comment:
you could put this.props.history.push inside of the .then block above, so you only move to the new page if the delete request succeeds